Index: .gradle/config.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#Wed Nov 05 18:22:18 ICT 2025\njava.home=C\\:\\\\Program Files\\\\Android\\\\Android Studio\\\\jbr\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/.gradle/config.properties b/.gradle/config.properties
--- a/.gradle/config.properties	(revision 65508b3b06109f73107841f86b19284f239ca52d)
+++ b/.gradle/config.properties	(date 1762486855188)
@@ -1,2 +1,2 @@
-#Wed Nov 05 18:22:18 ICT 2025
-java.home=C\:\\Program Files\\Android\\Android Studio\\jbr
+#Fri Nov 07 10:40:22 ICT 2025
+java.home=/Applications/Android Studio.app/Contents/jbr/Contents/Home
Index: local.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>## This file must *NOT* be checked into Version Control Systems,\n# as it contains information specific to your local configuration.\n#\n# Location of the SDK. This is only used by Gradle.\n# For customization when using a Version Control System, please read the\n# header note.\n#Tue Oct 28 12:09:00 ICT 2025\nsdk.dir=C\\:\\\\Users\\\\ADMIN\\\\AppData\\\\Local\\\\Android\\\\Sdk\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/local.properties b/local.properties
--- a/local.properties	(revision 65508b3b06109f73107841f86b19284f239ca52d)
+++ b/local.properties	(date 1762486860686)
@@ -4,5 +4,5 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Tue Oct 28 12:09:00 ICT 2025
-sdk.dir=C\:\\Users\\ADMIN\\AppData\\Local\\Android\\Sdk
+#Fri Nov 07 10:40:31 ICT 2025
+sdk.dir=/Users/hadi/Library/Android/sdk
